src/dvclive/live.py

```diff
@@ -191,6 +198,78 @@ def log_plot(self, name, labels, predictions, **kwargs):
         data.dump(val, self._step, **kwargs)
         logger.debug(f"Logged {name}")
 
+    def log_params(
+        self, params: Dict[str, ParamsLike], name: str = PARAMS_DEFAULT_NAME
+    ):
+        """Saves the given set of parameters (dict) to json"""
+        if self._resume:
+            logger.debug("resume=True, not removing existing parameters.")
+            return
+
+        output_dir = Path(self.dir) / self.PARAMS_DIR
+        output_dir.mkdir(exist_ok=True)
+        params_path = output_dir / (
+            f"{name}.json" if not name.endswith(".json") else name
+        )
+
+        if params_path.is_file():
+            with open(params_path, "r", encoding="utf-8") as fobj:
+                loaded_params = json.load(fobj)
```

Comment:
(Product) Should we load the existing params? I think we could just treat as the other things we log and overwrite unless `resume` is used

Reply:
This depends on how we want to handle `log_param`/`log_params`. If using `log_param(name, val)` I would expect this to append the parameter to an existing file. Using `log_params` could avoid loading the parameters, although it could be confusing to have different behaviors. 

Note that the params directory is always cleared on dvclive's init and we are not dumping parameters when `resume==True`

Reply:
Changed this slightly, now skipping `log_param/log_params` when `resume==True an step > 0`